Formula/postgis.rb

```diff
@@ -56,16 +62,22 @@ def install
     system "./configure", *args
     system "make"
 
+    # Install to a staging directory to circumvent the hardcoded install paths
+    # set by the PGXS makefiles.
     mkdir "stage"
     system "make", "install", "DESTDIR=#{buildpath}/stage"
 
-    bin.install Dir["stage/**/bin/*"]
-    lib.install Dir["stage/**/lib/*"]
-    include.install Dir["stage/**/include/*"]
-    (doc/"postgresql/extension").install Dir["stage/**/share/doc/postgresql/extension/*"]
-    (share/"postgresql/extension").install Dir["stage/**/share/postgresql/extension/*"]
-    pkgshare.install Dir["stage/**/contrib/postgis-*/*"]
-    (share/"postgis_topology").install Dir["stage/**/contrib/postgis_topology-*/*"]
+    # Because the Linux prefix is in a hidden folder (.linuxbrew), ** will not see it.
+    # Instead **/.*/** must be used to recurse into the stage directory
+    glob_string = OS.mac? ? "**" : "**/.*/**"
```

Comment:
Could we do something like this instead?

https://github.com/Homebrew/homebrew-core/blob/3398a5ff801b0aff0675b60c12d599e8f9629a9f/Formula/citus.rb#L38-L41

Reply:
Interesting idea.  It's a little bit more complicated because some things end up being installed in `stage/HOMEBREW_PREFIX` and others in `stage/prefix` (where `prefix` is the keg prefix).  However the general principle should still work.  I'll test this and push an update if I get it to work.